Assets/SEE/UI/SettingsMenu.cs

```diff
@@ -49,6 +51,31 @@ public class SettingsMenu : PlatformDependentComponent
         /// </summary>
         protected override void StartDesktop()
         {
+            string path = Application.dataPath + "/StreamingAssets/KeyBindings.json";
```

Comment:
Why are lines 54-78 not contained in a method of their own? And this method should belong to class `KeyBindings`. That class should have a load and save method. How the key bindings are stored must be a secret of this class.